rest-servlet/src/main/java/org/hawkular/inventory/rest/RestRelationships.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hawkular.inventory.rest;
+
+import com.wordnik.swagger.annotations.Api;
+import com.wordnik.swagger.annotations.ApiOperation;
+import com.wordnik.swagger.annotations.ApiResponse;
+import com.wordnik.swagger.annotations.ApiResponses;
+import org.hawkular.inventory.api.Inventory;
+import org.hawkular.inventory.api.Relationships;
+import org.hawkular.inventory.api.filters.RelationFilter;
+import org.hawkular.inventory.api.filters.RelationWith;
+import org.hawkular.inventory.api.model.Entity;
+import org.hawkular.inventory.api.model.Relationship;
+import org.hawkular.inventory.rest.json.ApiError;
+import org.hawkular.inventory.rest.json.RelationshipDeserializer;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import java.util.ArrayList;
+import java.util.List;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+/**
+ * @author jkremser
+ * @since 0.0.2
+ */
+@Path("/relationships")
+@Produces(APPLICATION_JSON)
+@Consumes(APPLICATION_JSON)
+@Api(value = "/relationships", description = "REST service for relationships")
+public class RestRelationships {
```

Comment:
I thought edges have their unique IDs?

Btw. I think it would be worth bringing this format to the dev list for discussion. While I think it is great that we are re-using a standard, I am not sure if we are supporting all the scenarios that we should in an ideal fashion. Or rather I am not even sure about the scenarios and queries that we need to support, so a discussion on the dev list I think would be beneficial.